docs/user_guide/git_repository.rst

```diff
@@ -0,0 +1,105 @@
+.. _git_repository:
+
+Testbenches Git repository
+===============================================================================
+
+This repository contains testbenches and verification components for system level
+projects or components connected at block level from the HDL repository.
+
+This repository is not a stand alone one. It must be cloned or linked as a
+submodule inside the HDL repository you want to test.
+
+All the **HDL** sources can be found in the following git repository:
+:git-hdl:`/`
+
+All the **Testbenches** sources can be found in the following git repository:
+:git-testbenches:`/`
+
+We assume that the user is familiar with `git <https://git-scm.com/>`__.
+Knows how to
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the repository, how to check its
+`status <https://git-scm.com/docs/git-status>`__ or how to
+`switch <https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging>`__
+between branches.
+
+.. note::
+
+   A basic git knowledge is required in order to work with these source files,
+   if you do not have any, don't worry!
+
+   There are a lot of great resources and tutorials about git all over the
+   `web <http://lmgtfy.com/?q=git+tutorial>`__.
+
+If you want to pull down the sources as soon as possible, just do the
+following few steps:
+
+#. Install Git from `here <https://git-scm.com/>`__
+#. Open up Git bash, change your current directory to a place where you
+   want to keep the hdl source
+#. Clone the repository using
+   `these <https://help.github.com/articles/cloning-a-repository/>`__
+   instructions
+
+Folder structure
+-------------------------------------------------------------------------------
+
+The root of the Testbenches repository has the following structure:
```

Comment:
Is someone assigned to implement the new structure yet?

Reply:
@IstvanZsSzekely will restructure the repo.

Reply:
The restructuring of the repository will occur once we finalize it.

Reply:
Should I undo this restructuring from the doc and update it later?

Reply:
I would leave as is, with the new structure